### DISCUSSION TITLE
Hide Update Notification

### DISCUSSION THREAD

**Discussion by MarioZVGL** [d2phap/ImageGlass#1745] [General] [answered]
Hi, how do I hide the update notifiation on imaglass for my users? We are working on multi session Terminal Server. 

![image](https://github.com/d2phap/ImageGlass/assets/129234463/ae07a197-195e-4ed3-9581-19b83483ae7e)

**Comment by d2phap** [comment, marked as answer]
You can create `igconfig.admin.xml` and set `AutoUpdate` to `"0"`. Refer to https://imageglass.org/docs/app-configs for more details about app configs
